examples/notebooks/beam-ml/run_inference_multi_model.ipynb

```diff
@@ -65,8 +64,8 @@
     {
       "cell_type": "markdown",
       "source": [
-        "A single machine learning model might not be the right solution for your task. Often, machine learning model tasks involve aggregating mutliple models together to produce one optimal predictive model and to boost performance. \n",
-        " \n",
+        "A single machine learning model might not be the right solution for your task. Often, machine learning model tasks involve aggregating mutliple models together to produce one optimal predictive model and to boost performance.\n",
+        "\n",
         "\n",
         "This notebook shows how to implement a cascade model in Apache Beam using the [RunInference API](https://beam.apache.org/documentation/sdks/python-machine-learning/). The RunInference API enables you to run your Beam transforms as part of your pipeline for optimal machine learning inference.\n",
```

Comment:
Can we link to https://beam.apache.org/documentation/sdks/python-machine-learning/#why-use-the-runinference-api?

Reply:
Good idea, linked to the main docs landing page instead though because I think its a better general starting place.